cuda_core/docs/source/release/0.5.0-notes.rst

```diff
@@ -0,0 +1,72 @@
+.. SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. currentmodule:: cuda.core.experimental
+
+``cuda.core`` 0.5.0 Release Notes
+=================================
+
+
+Highlights
+----------
+
+- Added memory mangment support (allocation, deallocation, copy, and fill) for CUDA graphs.
```

Comment:
```suggestion
- Added memory management support (allocation, deallocation, copy, and fill) for CUDA graphs.
```

Reply:
good catch, IIRC Sphinx has a spellcheck plugin, we should turn it on